### ISSUE TITLE
MessageInputView unable to disable the attach and lightning button

### ISSUE THREAD

**Issue by jonsung89** [GetStream/stream-chat-android#1446]
**Describe the bug**
I'm trying to disable & remove the attachment button and the lightning button from MessageInputView. 
When I first open the chat screen, the attachment button and the lightning button are NOT shown; however, when I start typing in the input view, the two buttons appear and stay visible. 

The only listener I have for the MessageInputView is the `setSendMessageHandler()` to just use `sendMessage()` to post the `messageText`

This is what I have for `MessageInputView` in my layout xml file

```
<androidx.cardview.widget.CardView
            android:id="@+id/messageInputLayout"
            android:layout_width="0dp"
            android:layout_height="wrap_content"
            android:layout_margin="@dimen/stream_margin_small"
            app:cardPreventCornerOverlap="false"
            app:cardCornerRadius="10dp"
            app:layout_constraintVertical_bias="1"
            app:layout_constraintTop_toBottomOf="@id/messagesList"
            app:layout_constraintBottom_toBottomOf="parent"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintStart_toStartOf="parent">

            <io.getstream.chat.android.ui.message.input.MessageInputView
                android:id="@+id/messageInputView"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                app:streamUiAttachButtonEnabled="false"
                app:streamUiLightningButtonEnabled="false"
                tools:layout_height="30dp"
                />
</androidx.cardview.widget.CardView>
```

**SDK version**
- 4.5.5

**Expected behavior**
With both value for `streamUiLightningButtonEnabled` and `streamUiAttachButtonEnabled` set to false, both buttons should not appear in any case.

**Device:**
 - Samsung A71 & Pixel 4XL

**Screenshots**
[screenshot 1](https://www.dropbox.com/s/t8qtmjvusz09k98/Screen%20Shot%202021-02-20%20at%201.35.15%20AM.png?dl=0)
[screenshot 2](https://www.dropbox.com/s/2mfv3cahgfh2qrk/Screen%20Shot%202021-02-20%20at%201.36.04%20AM.png?dl=0)

![Screen Shot 2021-02-18 at 11 59 28 AM](https://user-images.githubusercontent.com/32793713/108588213-979fa500-731d-11eb-81b5-d4ddadcb5896.png)

**Comment by bychkovdmitry** [comment]
Hello @jonsung89, thanks for reporting this. The fix is here - [#1447](https://github.com/GetStream/stream-chat-android/pull/1447]). Will be shipped in the next release.

**Comment by jonsung89** [comment]
Thank you!

**Comment by zsmb13** [comment]
This has been released in the latest version, [4.6.0](https://github.com/GetStream/stream-chat-android/releases/tag/4.6.0).
